src/pip/_internal/req/req_install.py

```diff
@@ -445,7 +445,7 @@ def check_if_exists(self, use_user_site):
         try:
             self.satisfied_by = pkg_resources.get_distribution(str(no_marker))
         except pkg_resources.DistributionNotFound:
-            return False
+            pass
```

Comment:
nit: this could be a `return`

Reply:
You're right, I think that's better. Updated!